lib/render/achievement.php

```diff
@@ -49,7 +49,7 @@ function GetAchievementAndTooltipDiv(
     }
 
     if ($inclSmallBadge) {
-        $achNameAttr = htmlspecialchars($achName, ENT_QUOTES);
+        $achNameAttr = attributeEscape($achName);
```

Comment:
`htmlspecialchars` escapes a lot of things that the tooltip didn't want to unescape (like `&frac12;`). This just escapes quotes and double quotes.